paasta_tools/utils.py

```diff
@@ -997,8 +998,24 @@ def get_net(self) -> str:
         """
         return self.config_dict.get("net", "bridge")
 
+    def has_bulkdata(
+        self,
+    ) -> bool:
+        return self.config_dict.get("uses_bulkdata", True)
+
     def get_volumes(self, system_volumes: Sequence[DockerVolume]) -> List[DockerVolume]:
         volumes = list(system_volumes) + list(self.get_extra_volumes())
+        # add a dictionary to docker_volumes with a hostPath of /nail/bulkdata
+        # if the service uses bulkdata
+        if self.has_bulkdata():
+            # the volume has a mode of RO
```

Comment:
nit:
```suggestion
        # we used to add bulkdata as a default mount - but as part of the 
        # effort to deprecate the entire system, we're swapping to an opt-in
        # model so that we can shrink the blast radius of any changes
        if self.has_bulkdata():
            # bulkdata is mounted RO as the data is produced by another 
            # system and we want to ensure that there are no inadvertent 
            # changes by misbehaved code
```
is likely a better set of comments since they add some context as to what's going on :)